docs/architecture-notes.md

```diff
@@ -1,5 +1,7 @@
 # Architecture Notes
 
+**USDS Note:** *This was our original planning on what could be done to modernize the legacy OneApp.*
+
```

Comment:
Once we've finished the architecture one-pager, we might want to link it here @mikepif